src/DevYeah.LMS/DevYeah.LMS.BusinessTest/Mock/AccountRepositoryMocker.cs

```diff
@@ -18,6 +18,11 @@ public void Add(Account entity)
             _accounts.Add(entity);
         }
 
+        public int Count(Expression<Func<Account, bool>> expression)
+        {
+            throw new NotImplementedException();
+        }
+
         public void Delete(Account entity)
```

Comment:
all one-line code can use bodied-expression function / member  :   =>

Reply:
And the logic of :

`public Account Get(Guid key)`

can be converted to simpler Lambda expression.

Same as 

`public Account GetUniqueAccountByEmail(string email)`

Any array search should do like this.